plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -1078,10 +1081,11 @@ describe('PersonState.update()', () => {
             hub.statsd = { increment: jest.fn() } as any
         })
 
-        it('stops $identify if current distinct_id is illegal', async () => {
+        const illegalIds = Array.from(ILLEGAL_IDS.values())
+        it.each(illegalIds)('stops $identify if current distinct_id is illegal: `%s`', async (illegalId: string) => {
```

Comment:
let's keep using hardcoded illegal ids in the tests to 1) not have too many tests taking more time 2) make sure we didn't mess up with the const